plugin/inc/monthlyplan.class.php

```diff
@@ -342,11 +348,15 @@ function setPreviousMonth(month) {
                                     if (isset($enterprise['moved'])) {
                                         $style .= "font-style:italic;font-weight:bold;";
                                     }
+
+                                    if (isset($enterprise['movements_count']) && $enterprise['movements_count'] > 0) {
+                                        $style .= "background-color:#ffffcc;";
+                                    }
```

Comment:
1. I would rather use some built-in color (like "orange", "blue" etc)
2. 
```suggestion
                                    if ($enterprise['movements_count'] ?? 0> 0) {
                                        $style .= "background-color:#ffffcc;";
                                    }
```
Anyway, can itt happen, that `$enterprise['movements_count']` is not set?